src/__tests__/getLocationForJsonPath.spec.ts

```diff
@@ -63,4 +64,26 @@ describe('getLocationForJsonPath', () => {
       });
     });
   });
+
+  describe('spectral bug #170 fixture', () => {
+    const result = parseWithPointers(spectral170);
+
+    test.each`
+      start       | end         | path
```

Comment:
Do we need each if its just a single test?

Reply:
You are right, we don't need `test.each`.
I wrote tests before I implement an actual fix, therefore I thought initially I might need to have a few more tests but turned out it's not really needed.
Changed!